src/bridge/rnastructure.h

```diff
@@ -47,6 +51,26 @@ class RNAstructure : public RnaPackage {
   std::unique_ptr<datatable> data_;
   bool use_lyngso_;
 
+  struct PartitionFunctionState {
```

Comment:
Can you put this in an anonymous namespace in the implementation file?

Reply:
Done.